proto/wippersnapper/pwm/v1/pwm.proto

```diff
@@ -6,10 +6,39 @@ package wippersnapper.pwm.v1;
 import "nanopb/nanopb.proto";
 
 /**
-* WriteDutyCycle represents a request to write a duty cycle to a pin with a frequency (fixed).
+* AttachRequest represents a request to a device to attach/allocate a PWM pin.
+* On ESP32 Arduino, this will "attach" a pin to a LEDC channel/timer group.
+* On non-ESP32 Arduino, this does nothing.
+*/
+message AttachRequest {
```

Comment:
I would like to match the style used by servo.proto here and call these: `PWMAttachRequest`, `PWMAttachResponse` and `PWMDetachResponse`, if possible.

Reply:
Resolved in https://github.com/adafruit/Wippersnapper_Protobuf/pull/107